### DISCUSSION TITLE
are 0.63.0-0 and 0.63.0-1 the stable releases?

### DISCUSSION THREAD

**Discussion by sudoexec** [outline/outline#3361] [General] [answered]
I see 0.63.0-0 and 0.63.0-1 in the release tags. But there isn't any release notes on Github or [Changes](https://www.getoutline.com/changelog).
Are 0.63.0-0 and 0.63.0-1 stable or beta release?

**Comment by tommoor** [comment, marked as answer]
No, they are prereleases as described by the version number.

**Reply by sudoexec** [reply]
Thanks. Sorry for my poor release number knowledge.

**Reply by almereyda** [reply]
I think GitHub even knows a way to tag certain releases as 'Pre-releases', as seen on https://github.com/logseq/logseq/releases/ for instance.

![Bildschirmfoto von 2022-04-13 00-01-32](https://user-images.githubusercontent.com/1645308/163062091-8c0bdbd5-eec2-4f3c-8b13-9dabd039eed4.png)

Which numbers would qualify for a release version, an even version number, which could hint at some kind of Semantic Versioning? If yes, there is possibly a chance here to add a note to README or someplace CONTRIBUTING or similar.
